### DISCUSSION TITLE
Manipulating Pleiotropy for MultiTrait Simulation

### DISCUSSION THREAD

**Discussion by B-Abbasi** [gaynorr/AlphaSimR#196] [Q&A] [answered]
AlphaSimR is simulating traits as Pleiotropic where some QTLS are effecting multiple traits, while others are only effecting a single trait. 
Can we control this Pleitropy in AlphaSimR to simulate different scenarios of Pleiotropy i.e Non-pleiotropy or all QTLs effecting all traits  ?

**Comment by gregorgorjanc** [comment, marked as answer]
You can control this by specifying uncorrelated traits - see example at https://github.com/HighlanderLab/jbancic_alphasimr_plants/blob/main/04_Features/multipleTraits.R

Or you can import effects (which you simulate according to your interest) - see example at https://github.com/HighlanderLab/jbancic_alphasimr_plants/blob/main/04_Features/importExternalHaplo.R

**Reply by B-Abbasi** [reply]
Thanks for sharing both options.
